### ISSUE TITLE
Guide doesnt list TV Episode Number and details

### ISSUE THREAD

**Issue by chamby90** [iptv-org/epg#332]
I am using a lot of the USA state specific EPG files to feed into Xteve/Plex and for the most part they work pretty well.  One of the issues I am having though is that tv shows are not being listed with their episode details and thus I cant use the DVR function in Plex because it thinks its a movie and will not save appropriately.  Is this something that could be fixed?

**Comment by chamby90** [comment]
and not sure if this has been posted elsewhere or deserves another thread... but I also noticed that if a 30 min TV show series is playing back to back, only the description from the first episode shows on both episodes in EPG.   Even though the 2nd 30 minute show is a different episode completely.

**Comment by freearhey** [comment]
I've added support for episode numbers and in the case of the directv.com guide, that information should show up tomorrow.

**Comment by chamby90** [comment]
thanks @freearhey !  Just to confirm... should this be showing in the state specific guides now as well and the TVTV guide?  My example I looked at today is the quest channel in the TVTV guide and it is showing every episode number as 01-20 (today s date - even though they are old episodes).

On a separate note - I was also previously using the Defy and TruReal channels in the Indiana guide but looks like those are not there anymore.  Should I submit a separate ticket about that?  Or are you still working on getting the state specific guides back together?

**Comment by freearhey** [comment]
The episode numbers in the guide from tvtv.us should also show up tomorrow. The same goes for the state specific guides.

As for the missing channels, it would really be better to create a separate issue so that if not me, someone else will add these channels back to the list.

**Comment by chamby90** [comment]
k thanks... @freearhey I will create a separate ticket about the missing channels and will check again on the episode numbers.

**Comment by chamby90** [comment]
@freearhey unless I am missing something, I am still seeing the episode numbers show as todays date for the Quest channel in the TVTV guide at least:
![image](https://user-images.githubusercontent.com/74013500/150536635-bfb3302c-97bd-4c34-af73-006551f61f52.png)

I was also curious if there is a field in the xml that controls when a show is deemed "new", or is that handled solely based on the date of the episode?  since every episode is showing today's date, they are all showing as new for me. 

You did mention the DirecTV guide in one of your previous posts, so i tried that too using TruTV and it does indeed have the episode number, but is still showing up as a "new" episode for me.  
![image](https://user-images.githubusercontent.com/74013500/150537772-646d2ab6-3282-4145-8c60-a2c484d0ca1b.png)

**Comment by Bodyjewels** [comment]
Looking for help loading epg in us into tivimate.. Put in perfect still will not load in..any advice newbie

**Comment by chamby90** [comment]
@freearhey I have finally come around to using the newer EPG solution and have generated an xmltv file using the tvtv.us guide and I'm back to having the same issue I had a few years ago .

![image](https://github.com/iptv-org/epg/assets/74013500/ac406441-6e4e-4610-b99c-84b6f23e67b3)

Not having the episode details/date plus the images are not pulling across.  Maybe Im not doing something right again, but not sure if you know something quickly that would explain what im seeing?

**Comment by chamby90** [comment]
@freearhey  - for what its worth, the same thing is happening with the Passport.com guide (except i am getting images with that one atleast).  Im guessing this is probably specific to how Plex is interpreting the files that get saved via xteve and then pushed to plex (unless im told this is a known issue for all xmltv generated files)

![image](https://github.com/iptv-org/epg/assets/74013500/2cf93b09-c1b2-4098-819d-d35530ad72c9)

**Comment by chamby90** [comment]
and then... because i was curious, instead of assigning the EPG in xteve and then sending the xteve EPG into Plex, I added the EPG passport created guide directly into plex and get a slightly different result:

![image](https://github.com/iptv-org/epg/assets/74013500/380ba160-0ef4-455c-bf6b-7521d7773970)

so if i interpret this correctly, both xteve and plex are unable to find the episode details correctly, the only difference is that xteve then adds the date as the episode before sending to plex

**Comment by Bodyjewels** [comment]
Is it possible to add the guide to t i v i m a t e this is where I have it loaded old lady here new to all of this but learning

**Comment by chamby90** [comment]
@Bodyjewels not sure if you are posing that question to me, but I personally am not familiar with how tivimate works.  But if you are able to reference an XML file on your own computer in that program than you would be able to use this method to create a guide.  This particular open issue is to talk through how to get the season and episode details to carry forward into xteve/plex.  Do you have an issue with that particular thing?

**Comment by PopeyeTheSai10r** [comment]
> Is it possible to add the guide to t i v i m a t e this is where I have it loaded old lady here new to all of this but learning

Yes, but it wouldn't just work. There's may be some manual linking needed. This should be it is own [discussion](https://github.com/orgs/iptv-org/discussions).

**Comment by chamby90** [comment]
@freearhey - just confirming that you opened this back up because you confirmed this is a legit issue right?  Just want to make sure I'm not doing something wrong or if the issue is more related to Plex or Xteve.

**Comment by chamby90** [comment]
checking in.... any update on if this is a resolved issue?  Id like to come back to using the EPG to combine a few guides for my channels, but in the meantime and just using the Plex EPG for one guide.

**Comment by freearhey** [comment]
To be honest I have already lost the thread of this discussion. Which specific site is currently missing an episode number?

**Comment by chamby90** [comment]
@freearhey - I had tried the tvtv and passport guides in the past and both of them will not display correct episode details in Plex using xteve.   If I call the EPG directly into Plex and bypass xteve it doesnt work either (as i thought maybe xteve was the problem).   That was my post in Dec of 2023.  Its been a while since I have tried to use the tvtv or passport guides so not sure if maybe that has been corrected at this point?

**Comment by freearhey** [comment]
I have added some code to the config that allows downloading the requested data, but unfortunately due to restrictions from tvtv.us we can't use it. Details: https://github.com/iptv-org/epg/pull/2668

But if something changes in the future we will definitely revisit this issue.
